samples/testing-frameworks/appium/client-side/python/tests/bitbar_biometrics_ios.py

```diff
@@ -15,13 +15,13 @@ def setUp(self):
         if self.bitbar_device == "":
             self._find_device("iOS")
 
-        self.desired_capabilities_cloud["bitbar_project"] = bitbar_project_name
-        self.desired_capabilities_cloud["bitbar_device"] = self.bitbar_device
         self.desired_capabilities_cloud["platformName"] = "iOS"
-        self.desired_capabilities_cloud["deviceName"] = "iPhone device"
-        self.desired_capabilities_cloud["bitbar_target"] = "ios"
-        self.desired_capabilities_cloud["automationName"] = automation_name
-        self.desired_capabilities_cloud["bitbar_biometricInstrumentation"] = True
+        self.desired_capabilities_cloud["bitbar:options"]["project"] = bitbar_project_name
+        self.desired_capabilities_cloud["bitbar:options"]["device"] = self.bitbar_device
+        self.desired_capabilities_cloud["bitbar:options"]["target"] = "ios"
+        self.desired_capabilities_cloud["bitbar:options"]["biometricInstrumentation"] = True
+        self.desired_capabilities_cloud["appium:automationName"] = automation_name
+        self.desired_capabilities_cloud["appium:deviceName"] = "iPhone device"
```

Comment:
For some reason removing `appium:deviceName` caused an error (`deviceName can't be blank`). According to [appium documentation](https://appium.io/docs/en/2.0/guides/caps/) that shouldn't happen assuming we're using `udid` to specify the device and I think we do.